CardsCashCasino/Manager/CardManager.cs

```diff
@@ -25,6 +77,36 @@ public void Draw(SpriteBatch spriteBatch)
         {
 
         }
+
+        /// <summary>
+        /// Adds card to the discard queue. 
+        /// Caller must guarantee that card has been removed from whatever collection it was in prior.
+        /// </summary>
+        public void Discard(Card card)
+        {
+            _discard.Add(card);
+        }
+
+        /// <summary>
+        /// Moves all cards from discard into the deck, and randomizes the deck.  
+        /// 
+        /// Guarantees that discard's old size + deck's old size = deck's new size, and that discard's new size = 0.
+        /// </summary>
+        public void Recycle() 
```

Comment:
This should be recycling the discard pile into the Cards list since that is what is going to be used in the games, correct? _deck is being used to efficiently populate the Cards list if I'm understanding the intent behind it correctly.

Reply:
Just checked and yeah, you're right. @jacobwilki01 why is the deck called Cards and the list of cards called _deck? Shouldn't we do it the other way around?